onnx_tf/handlers/backend/onehot.py

```diff
@@ -10,6 +11,21 @@
 @tf_func(tf.one_hot)
 class OneHot(BackendHandler):
 
+  @classmethod
+  def args_check(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    indices = tensor_dict[node.inputs[0]]
+    depth = tensor_dict[node.inputs[1]][0]
```

Comment:
Maybe depth = tensor_dict[node.inputs[1]] because it is the true input. It would be consistent with indices as well. The dtype check should work this way.

Reply:
@chinhuang007 updated, please review again thanks

Reply:
I am okay with using the variable name "depth_array" but it is not accurate with the API. The input is depth, type is vector, as defined as // Input 'depth' must be a single-element vector in  https://github.com/onnx/onnx/blob/ceaa5da720418b2b705f6066d82c38fe89694f12/onnx/defs/tensor/defs.cc#L1525